src/app/components/core/south/south-service-modal/south-service-modal.component.ts

```diff
@@ -527,9 +480,108 @@ export class SouthServiceModalComponent implements OnInit, OnChanges {
     this.isFilterDeleted = false;
     this.deletedFilterPipeline = [];
     if (this.applicationTagClicked) {
-      this.isWizard = this.applicationTagClicked;
+      this.isAddFilterWizard = this.applicationTagClicked;
       return;
     }
     this.toggleModal(false);
   }
+
+  /**
+   * Get edited south service configuration from show configuration page
+   * @param changedConfiguration changed configuration of a selected plugin
+   */
+  getChangedConfig(changedConfiguration: any) {
+    this.changedConfig = this.configurationControlService.getChangedConfiguration(changedConfiguration, this.pluginConfiguration);
+  }
+
+  /**
+  * Get edited south service advance configuration
```

Comment:
What about the security config?

Reply:
Common method for advanced and security configuration.